File: Day Four/Java/After Break/TaskThree.java
import java.util.Scanner;

public class TaskThree {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int count = 0;
        int sum = 0;
        int average = 0;
        int result= 0;
       
        while (count < 10) {
            System.out.print("Enter a score: ");
            int userInput = scanner.nextInt();
            
            sum += userInput;
            average = sum / count;
            result = sum + average;             
            count++;

        }

        
        {  System.out.println("The average of 10 scores is: " +  average);

        }

    }
}

File: Day Four/Python/After Break/Task_Three.py

sum_ = 0
count = 0
average = 0

while count < 10:
    number = int(input(f"Enter score {count + 1}: "))
    sum_ += number 
    count += 1
    average = count / sum_

print("The average of the 10 scores is:", average)
print("The som of 10 scores is:", sum_)

